### ISSUE TITLE
#108 Chat

### ISSUE THREAD

**Issue by SENOSuit** [HUNTERS1984/tenposs#16]
## Content

● エンドユーザーとのチャット機能
- 基本はLINEやSkypeを踏襲
  ● チャットの通知機能

**Comment by SENOSuit** [comment]
### 藤吉さん

＊ユーザーごとの個別なのか、１つのチャットタイムラインでやり取りするのか？

＊初期はLINE＠の連携でやるか、独自でやるか？

＊個別の場合デザインはどのように見せるのか？

＊チャットが入った場合、すぐにお店側が気付くためのアラートなども用意が必要。

### 酒井

ユーザーごとの個別でやりたいところですが、
LINE＠だと全ユーザーになってしまうのでしたっけ？

**Comment by ryo2mattock** [comment]
LINE@でも個別でいけると思います。なので初期はシンプルにLINE@に遷移で良いかと思います。

**Comment by SENOSuit** [comment]
了解です。
では、 LINE@に遷移でいきましょう。

**Comment by ryo2mattock** [comment]
はい、 LINE@に遷移でお願いします。

**Comment by SENOSuit** [comment]
その場合ですが、admin側でのChatメニューはサイドメニューからLINE@へ遷移という形でいいですかね？

**Comment by ryo2mattock** [comment]
そうですね。これもLINE@アカウントを設定で決めておき遷移されるというのがAPIで実現出来ればそれで良いと思います。あとはAPI取得の工数とコスト次第ですかね。

**Comment by SENOSuit** [comment]
> そうですね。これもLINE@アカウントを設定で決めておき遷移されるというのがAPIで実現出来ればそれで良いと思います。あとはAPI取得の工数とコスト次第ですかね。

これ、そんなに工数かからないと思います。
ただ、LINE@のアカウントにログインすればいいと思うので、API側もそんなに難しくはないかと。

**Comment by ryo2mattock** [comment]
なるほど、軽ければ実装で重ければ別途検討と再協議しましょうか。

**Comment by SENOSuit** [comment]
はい、そうしましょう。

**Comment by ryo2mattock** [comment]
はい。
